packages/use-sync-external-store/package.json

```diff
@@ -17,6 +17,20 @@
     "shim/",
     "cjs/"
   ],
+  "exports": {
+    ".": "./index.js",
+    "./index": "./index.js",
+    "./index.native": "./index.native.js",
+    "./with-selector": "./with-selector.js",
+    "./shim": "./shim/index.js",
+    "./shim/index": "./shim/index.js",
+    "./shim/index.native": "./shim/index.native.js",
+    "./shim/with-selector": "./shim/with-selector.js",
+    "./shim/*": "./shim/*",
+    "./cjs/*": "./cjs/*",
```

Comment:
Why do we need these?

Reply:
see comment below, I believe this was for backwards compatibility.